src/creates/updateIssue.ts

```diff
@@ -0,0 +1,240 @@
+import { Bundle, ZObject } from "zapier-platform-core";
+import { fetchFromLinear } from "../fetchFromLinear";
+import { omitBy, uniq } from "lodash";
+
+interface IssueUpdateResponse {
+  data?: {
+    issueUpdate: {
+      issue: {
+        id: string;
+        title: string;
+        url: string;
+        identifier: string;
+      };
+      success: boolean;
+    };
+  };
+  errors?: {
+    message: string;
+    extensions?: {
+      userPresentableMessage?: string;
+    };
+  }[];
+}
+
+interface IssueResponse {
+  data: { issue: { labelIds: string[] } };
+}
+
+const updateIssueRequest = async (z: ZObject, bundle: Bundle) => {
+  if (!bundle.inputData.issueIdToUpdate) {
```

Comment:
Why not just `issueId` btw?

Reply:
Just to differentiate since `issueId` is already used in another trigger I didn't want to create any accidental side effects